### PR TITLE
Feature/#34 config api

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/apps/common/models.py [hunk 1]
```diff
@@ -36,6 +25,7 @@ class Point(TimeStampedModel):
         decimal_places=5,
         validators=[MinValueValidator(Decimal(-180)), MaxValueValidator(Decimal(180))]
     )
+    # TODO: find out what UNICEF understands exactly under this name (admin_level_1)
```
Comment (elyk) [inline backend/unpp_api/apps/common/models.py]: Should be clarified given last convo: https://en.wikipedia.org/wiki/Category:First-level_administrative_country_subdivisions

File: backend/unpp_api/apps/common/permissions.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from rest_framework.permissions import BasePermission
+
+
+class IsAuthenticated(BasePermission):
```
Comment (elyk) [inline backend/unpp_api/apps/common/permissions.py]: Any reason we're creating our own class here vs. using out of the box w/ DRF?

http://www.django-rest-framework.org/api-guide/permissions/#isauthenticated

Comment (lkolacz) [inline backend/unpp_api/apps/common/permissions.py]: I thought we could extend this class, but we can use DRF example, and create dedicated like can read ... ;-) .. will remove it !

File: backend/unpp_api/apps/common/models.py (hunk 1 above)
Comment (lkolacz) [inline backend/unpp_api/apps/common/models.py]: in different branch I explain in example this part .. it's old comment .. in next branches (PR) changes will come .. please ignore this ..

**Comment by lkolacz** [comment]
I'm merging this since I give feedback to Kyle comment and bring updates in other fields.

**Pull request merged**